app/views/person_mailer/presentation_notification.html.erb

```diff
@@ -0,0 +1,9 @@
+<DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <p>Hello world</p>
```

Comment:
I think we can add actual content here since we're just stubbing out the model that we're going to be getting anyways.